data/mods/Magiclysm/items/metals.json

```
@@ -0,0 +1,40 @@
+[
+  {
+    "id": "orichalcum_lump",
+    "type": "AMMO",
+    "category": "spare_parts",
+    "name": { "str": "lump of orichalcum", "str_pl": "lumps of orichalcum" },
```

```suggestion
    "name": "lump of orichalcum",
```
Just found out that plural names are actually not allowed in the string extraction script for the `AMMO` type.